src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,16 @@
 import './MovieList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      !movie
```

Comment:
movie here is just array item in movies, I am not sure if it can happen (!movie)